src/dotcms-js/core/util/http.service.ts

```diff
@@ -0,0 +1,135 @@
+import {Injectable} from '@angular/core';
+import {Http, Headers, Response, RequestMethod, RequestOptions} from '@angular/http';
+import {Observable} from 'rxjs';
+
+import 'rxjs/add/operator/map';
```

Comment:
I don't think you need to import this... we use it in other places without importing them.